Layout/g4 sp 95/apep

itsapep opened the pull request:
Beautify & re design front-end elements (Make it more interactive)
I don't think lottie animations are needed as the main idea of the website is monochrome so I only add pink accent to some of the main pages

itsapep committed:
[add] create About Page

itsapep committed:
[edit] change animation for loading modal

itsapep committed:
[add] create midtrans snap implementation

[Payment - FE] Implement midtrans popup payment confirmation on our payment confirmation page

itsapep committed:
[edit] change navbar behaviour

add on hover dropdown show, relayout container, etc.

itsapep committed:
[edit] beautify homepage 

add glow on hover for service card and smooth page transition

itsapep committed:
[edit] beautify sign in page

itsapep committed:
[edit] fix naming on css file

itsapep committed:
Merge branch 'trial-apep/aesthethics/homepage' into trial-apep/aesthethics/general

itsapep committed:
Merge branch 'trial-apep/aesthethics/sign-in' into trial-apep/aesthethics/general

itsapep committed:
Merge branch 'trial-apep/aesthethics/navbar' into trial-apep/aesthethics/general

itsapep committed:
[edit] fix files after merges

itsapep committed:
Merge branch 'layout/G4SP-95/apep' into trial-apep/aesthethics/general

itsapep committed:
[edit] beautify request-list

itsapep committed:
[edit] beautify purchase-list

itsapep committed:
[edit] fix html web name and icon

itsapep committed:
Merge branch 'trial-apep/aesthethics/request-list' into trial-apep/aesthethics/general

itsapep committed:
Merge branch 'trial-apep/aesthethics/purchase-list' into trial-apep/aesthethics/general

itsapep committed:
Merge branch 'trial-apep/aesthethics/general' into layout/G4SP-95/apep

PandaNoob13 reviewed (changes requested):
* Good job yurham! Please change the gradient colour to gray so it still match the colour palette we designed!

itsapep committed:
[edit] fix pages colour gradient

itsapep committed:
[edit] add transition on sign up view

itsapep committed:
[edit] fix some beauty

tidy up sign up view, change loading to swiping-diamond, change margin to padding for pages

PandaNoob13 reviewed (approved):
* Good job yurham!
